### PR TITLE
Update GAP/GATT test cases

### PR DESCRIPTION
**Pull request by szymon-czapracki**
This updates some GAP/GATT test cases to use generic wid handler.

### CODE REVIEW

**Comment by szymon-czapracki** [comment]
Please, do not merge.

File: ptsprojects/zephyr/gattc_wid.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    try:
+        handler = getattr(module, "hdl_wid_%d" % wid)
+        return handler(description)
+    except AttributeError as e:
+        return gatt_wid_hdl(wid, description, test_case_name)
+
+
+def hdl_wid_24(desc):
+    return btp.verify_description(desc)
```
Comment (MariuszSkamra) [inline ptsprojects/zephyr/gattc_wid.py]: new line missing at the end of this file

File: ptsprojects/zephyr/gap.py [hunk 2]
```diff
@@ -71,6 +71,10 @@ class CHAR:
 
 iut_device_name = 'Tester'
 iut_manufacturer_data = 'ABCD'
+iut_appearance = '1111'
```
Comment (MariuszSkamra) [inline ptsprojects/zephyr/gap.py]: hmm, this appearance could be set to something more meaningful. Check Bluetooth assigned numbers

File: ptsprojects/zephyr/gap.py [hunk 3]
```diff
@@ -71,6 +71,10 @@ class CHAR:
...
+iut_svc_data = '1111'
+iut_flags = '11'
+iut_svcs = '1111'
```
Comment (MariuszSkamra) [inline ptsprojects/zephyr/gap.py]: This value reflects the service that is actually in GATT database. Maybe it's good to change this value to e.g. GAP service UUID

File: ptsprojects/zephyr/gap_wid.py [hunk 4]
```diff
@@ -89,16 +89,91 @@ def hdl_wid_14(desc):
...
+
+    ad = []
+
+    if stack.gap.name:
+        ad.append((AdType.uuid16_some, stack.gap.svcs))
+
+    btp.gap_adv_ind_on(ad=ad)
+
+    return True
+
```
Comment (MariuszSkamra) [inline ptsprojects/zephyr/gap_wid.py]: add extra new line

File: ptsprojects/zephyr/gap_wid.py [hunk 5]
```diff
@@ -438,8 +686,20 @@ def hdl_wid_178(desc):
...
 def hdl_wid_1002(desc):
     stack = get_stack()
     passkey = stack.gap.passkey.data
     stack.gap.passkey.data = None
     return passkey
+
+
+def hdl_wid_2142(desc):
+    btp.gap_conn()
+    return True
```
Comment (MariuszSkamra) [inline ptsprojects/zephyr/gap_wid.py]: no new line at the end

File: ptsprojects/zephyr/gattc_wid.py (hunk 1 above)
Comment (szymon-czapracki) [reply ptsprojects/zephyr/gattc_wid.py]: Done

File: ptsprojects/zephyr/gap_wid.py (hunk 4 above)
Comment (szymon-czapracki) [reply ptsprojects/zephyr/gap_wid.py]: Done

File: ptsprojects/zephyr/gap_wid.py (hunk 5 above)
Comment (szymon-czapracki) [reply ptsprojects/zephyr/gap_wid.py]: Done

**Pull request merged**